frontend/src/components/supp-allowances/TransportationAllowance.vue

```diff
@@ -206,12 +233,20 @@ export default {
     isWarningDisplayed() {
       return this.models?.some((el) => isApplicationLocked(el?.statusCode))
     },
+    showBackDatePicker() {
+      const today = new Date()
+      return today > new Date(this.startDate)
```

Comment:
should we set today and new Date(this.startDate) to start of the day before we compare them? otherwise, it will include hour/minute/second in the comparison --> always show date picker.

Reply:
done